framework/include/bcs/IntegratedBC.h

```diff
@@ -67,6 +67,22 @@ class IntegratedBC : public IntegratedBCBase, public MooseVariableInterface<Real
     return computeQpOffDiagJacobian(jvar);
   }
 
+  /**
+   * Put necessary evaluations depending on qp but independent on test functions here
```

Comment:
```suggestion
   * Insertion point for evaluations that depend on qp but are independent of the test functions.
```

Reply:
Trying to make some of this consistent. Would you mind changing the doc string for the methods that follow in a similar manner?

If you have the time - I'd also appreciate copying this to DGKernel. I think I missed this last time.

Reply:
ok